src/remote_action_middleware.js

```diff
@@ -1,6 +1,15 @@
+import constants from '../src/constants';
+
 export default (socket) => (store) => (next) => (action) => {
   if (action.meta && action.meta.remote) {
     socket.emit(action.type, action);
   }
+
+  if(action.onSuccess) {
+    const { type, onSuccess } = action.onSuccess;
+
+    socket.on(type, onSuccess);
```

Comment:
Oh I see.. Would we be attaching a new listener each time we create an action with an `onSuccess` prop? Are there any concerns with that?

Also, there are some conflicts. Otherwise, looks okay to me.